yarn-project/acir-simulator/src/client/private_execution.ts

```diff
@@ -163,6 +163,13 @@ export class PrivateFunctionExecution {
 
         return toAcvmCallPrivateStackItem(childExecutionResult.callStackItem);
       },
+      getL1ToL2Message: ([msgKey]: ACVMField[]) => this.context.getL1ToL2Message(fromACVMField(msgKey)),
+
+      emitData: ([data]: ACVMField[]) => {
+        // eslint-disable-next-line
+        console.log(data);
```

Comment:
👀 Is emitting the events just logging it to console? Or is this just stale?

Reply:
Ok, ill rename it to debug_log?